dockerfiles/header_sync/Dockerfile

```diff
@@ -38,7 +34,7 @@ RUN ln -sf /dev/stdout /app/vulcanizedb.log
 COPY --from=builder /vulcanizedb/vulcanizedb .
 COPY --from=builder /vulcanizedb/dockerfiles/header_sync/startup_script.sh .
 COPY --from=builder /vulcanizedb/db/migrations/* db/migrations/
-COPY --from=builder /go/src/github.com/pressly/goose/cmd/goose/goose goose
+COPY --from=builder /go/bin/goose goose
```

Comment:
Just for due diligence - have you been able to build and run this image locally (e.g. not using `header_sync:latest` from dockerhub)?

Reply:
Yeah I've run it. See above - header_sync:latest is my local image name. The dockerhub version is makerdao/header_sync.

Besides the latest and greatest in dockerhub wouldn't work this way, since goose doesn't take a --table param.